### PR TITLE
add ci script for infernece

### PR DESCRIPTION
**Pull request by LianxinGao**
as title

### CODE REVIEW

File: ci_script.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    model = PeftModel.from_pretrained(model, lora_weights_path, torch_dtype=torch.float16)
+    input_ids = tokenizer(prompt, return_tensors="pt").input_ids.to(device)
+    with torch.inference_mode():
+        outputs = model.generate(
+            input_ids=input_ids,
+            max_new_tokens=100, do_sample=True, top_p=0.9, temperature=0.5
+        )
+        output = tokenizer.batch_decode(outputs.detach().cpu().numpy(), skip_special_tokens=True)[0][len(prompt):]
+
+        print(f"Prompt:\n{prompt}\n")
```
Comment (merlintang) [inline ci_script.py:35]: do we need to add the test here? assert something?

File: .github/workflows/run_on_dev.yml [hunk 2]
```diff
@@ -0,0 +1,80 @@
+name: Test LLM Models on Dev
+
+on:
+  push:
+    except:
+      branches:
+        - 'main'
```
Comment (mikecovlee) [inline .github/workflows/run_on_dev.yml:7]: still, main branch

Comment (mikecovlee) [reply .github/workflows/run_on_dev.yml:7]: plz add dev branch

**Comment by yezhem** [comment]
image

File: .github/workflows/run_on_dev.yml (hunk 2 above)
Comment (mikecovlee) [reply .github/workflows/run_on_dev.yml:7]: Sorry, I saw it wrong. But can you limit to *_dev branches only？

Comment (yezhem) [reply .github/workflows/run_on_dev.yml:7]: why need a limit to *_dev?

Comment (LianxinGao) [reply .github/workflows/run_on_dev.yml:7]: > Sorry, I saw it wrong. But can you limit to *_dev branches only？

why need a limit to *_dev?

File: ci_script.py (hunk 1 above)
Comment (LianxinGao) [reply ci_script.py:35]: The output is random, we cannot assert an uncertain result...

File: .github/workflows/run_on_dev.yml (hunk 2 above)
Comment (mikecovlee) [reply .github/workflows/run_on_dev.yml:7]: I think limit to *_dev is more specific. But never mind, go for merge.

File: ci_script.py (hunk 1 above)
Comment (merlintang) [reply ci_script.py:35]: similar

**Review by yezhem — APPROVED** [review]

**Pull request merged**